requirements.in

```diff
@@ -0,0 +1,9 @@
+-f https://download.pytorch.org/whl/torch_stable.html
+
+numpy
+flask
+requests
+torch==1.9.0+cpu
+aitextgen
+python-dotenv
+black
```

Comment:
This is a new file related to pip-tools which you can read about here: https://github.com/jazzband/pip-tools . Let's just talk about this part in person sometime, cause it's kind of hard to explain it here.

In general, pip-tools is just an alternative to pipenv that is a little more sane. All this pip-tools stuff is unrelated to the reddit stuff, but I was just kind of trying to organize stuff a little bit.

The one important change I added here is that I switched it to using the `cpu` version of torch which is about 7 times smaller disk-space-wise and I'm hoping it takes up less memory also when running